openshift/api-deploy.yml

```diff
@@ -64,11 +64,11 @@ objects:
           name: ${NAME}-${COMPONENT}${SUFFIX}-${API_VERSION}
           resources:
             limits:
-              cpu: 80m
-              memory: 96Mi
+              cpu: 400m
```

Comment:
How have you come up with these limits Amlan? Are they based on the existing API (Reviews) and how much it requires?

Reply:
These limits are based on some existing APIs from Digital forms project...I added some more on top of the existing numbers...